conftest.py

```diff
@@ -225,11 +225,16 @@ def pytest_runtest_setup(item: Item) -> None:
     1. Updates global config (`updated_global_config`)
     2. Adds `fail_if_missing_dependent_operators` fixture for Serverless tests.
     3. Adds fixtures to enable KServe/model mesh in DSC for model server tests.
+    4. Adds `fail_if_missing_authorino_operator` fixture for model registry tests.
     """
 
     BASIC_LOGGER.info(f"\n{separator(symbol_='-', val=item.name)}")
     BASIC_LOGGER.info(f"{separator(symbol_='-', val='SETUP')}")
 
+    # Add Authorino operator check for model registry tests
+    if "model_registry" in str(item.fspath):
+        item.fixturenames.insert(0, "fail_if_missing_authorino_operator")
+
```

Comment:
This is a hacky way. Is there any reason to not have `autouse=True`.  https://docs.pytest.org/en/stable/how-to/fixtures.html#autouse-fixtures-fixtures-you-don-t-have-to-request

Reply:
I've done the same as the other calls already here, which ensures this fixture is run for any test under model_registry but not for any other (which is what we want).

Reply:
I know model serving did it this way because of the complexity of their need, but our use case is simpler and can use autouse.